homeassistant/components/notify/html5.py

```diff
@@ -144,6 +157,21 @@ def _load_config(filename):
     return {}
 
 
+class HTML5ApplicationServerKeyView(HomeAssistantView):
+    """To fetch application_server_key from a browser."""
+
+    url = '/api/notify.html5/applicationServerKey'
```

Comment:
it is better change to 
```python
url = '/api/notify.html5/application.server.key'
```

Mixed up upper case and lower case in URL is not a good idea

Reply:
OK, used the same name as webpush subscribe property to be extra clear on what it is, but i can change it.